src/smdba/postgresqlgate.py

```diff
@@ -941,7 +941,7 @@ def do_system_check(self, *args, **params):
             changed = True
 
         # max_locks_per_transaction
-        if not conf.get('max_locks_per_transaction'):
+        if not conf.get('max_locks_per_transaction') or conf.get('max_locks_per_transaction') < '100':
```

Comment:
@dmacvicar because if it is already a value, it won't fix the problem (setting _no less than_ 100 locks), as I understand.